src/System.Runtime.InteropServices/tests/System.Runtime.InteropServices.Tests.csproj

```diff
@@ -141,5 +133,20 @@
     <Compile Include="System\Runtime\InteropServices\TypeLibTypeAttributeTests.cs" />
     <Compile Include="System\Runtime\InteropServices\TypeLibVarAttributeTests.cs" />
     <Compile Include="System\Runtime\InteropServices\TypeLibVersionAttributeTests.cs" />
+    <Compile Include="System\Runtime\InteropServices\Marshal\GetHINSTANCETests.cs" />
+    <Compile Include="System\Runtime\InteropServices\Marshal\ReadWrite\ByteTests.nonnetstandard.cs" />
```

Comment:
I guess we now have a new pattern, `nonnetstandard.cs`. I can't really think of a better one ulness maybe `notnetstandard.cs`.

Reply:
I know that we usually try to be inclusive, as opposed to exclusive naming-wise, but in this case, I couldn't think of a better one. I could go with notnetstandard.cs if you prefer as I am not inclined to any of the two.